src/frontend.js

```diff
@@ -5,6 +5,7 @@
 
 // import dependencies
 import { Accordion } from 'govuk-frontend';
+import { Details } from 'govuk-frontend';
 
 // get components
 /** Accordion **/
```

Comment:
Do you need to add Details initialization below as well?

Reply:
Done